$OpenBSD: patch-Tests_Unit_GNUmakefile,v 1.10 2021/09/14 12:21:58 sebastia Exp $
Do not hardcode libobjc library name, ask GNUstep what to use.
Index: Tests/Unit/GNUmakefile
--- Tests/Unit/GNUmakefile.orig
+++ Tests/Unit/GNUmakefile
@@ -45,8 +45,8 @@ $(TEST_TOOL)_CPPFLAGS += \
 	-Wall -D_GNU_SOURCE -I../../SOPE/ -I../../SoObjects/ -I../../UI/ -I../../OpenChange
 
 ADDITIONAL_LIB_DIRS += \
-	-L../../SoObjects/SOGo/SOGo.framework/Versions/Current/sogo -L../../SOPE/NGCards/obj -L../../SOPE/GDLContentStore/obj -lSOGo -lNGMime -lNGCards -lGDLContentStore -lNGExtensions -lSBJson -lobjc \
-	-L/usr/local/lib  -lSaxObjC -lNGStreams \
+	-L../../SoObjects/SOGo/SOGo.framework/Versions/Current/sogo -L../../SOPE/NGCards/obj -L../../SOPE/GDLContentStore/obj -lSOGo -lNGMime -lNGCards -lGDLContentStore -lNGExtensions -lSBJson \
+	-L/usr/local/lib  `gnustep-config --objc-libs` -lSaxObjC -lNGStreams \
 	-Wl,-rpath,../../SoObjects/SOGo/SOGo.framework/Versions/Current/sogo -Wl,-rpath,../../SOPE/NGCards/obj -Wl,-rpath,../../SOPE/GDLContentStore/obj
 ADDITIONAL_LDFLAGS += -Wl,--no-as-needed
 
